### ISSUE TITLE
arrow function doesn't get replaced with ecmascript

### ISSUE THREAD

**Issue by balazsbencs-attrecto** [VeliovGroup/meteor-autoform-file#41]
Running meteor with --production flag or deploying to Galaxy, one arrow function in fileUpload.js doesn't get replaced with ecmascript and es5-shim installed, and it breaks compatibility in Internet Explorer.
Every other code part works fine, this is the only arrow method that doesn't get replaced.

**lib/client/fileUpload.js**
starting with line 44
`
this.collectionName = () => {
    return this.data.atts.collection;
  };
`

**Comment by balazsbencs-attrecto** [comment]
Some weird stuff happened: I wanted to poke around see if I can break it with a custom file upload template that I copied from the original.
I tried to store the function in a variable, and I only put the variable above that line, and it worked. Not just this variable was transpiled correctly, but the other one too. When I removed the new code, it was failing again.

```
  let collectionName = () => {
    return this.collection.collectionName();
  };

  this.collectionName = () => {
    return this.collection.collectionName();
  };
```

**Comment by dr-dimitru** [comment]
@balazsbencs-attrecto thank you a lot for reporting about this.
Sorry for delay, fix is published in [`v2.1.0`](https://github.com/VeliovGroup/meteor-autoform-file/releases/tag/2.1.0). Please upgrade to the latest release and let me know how it works for you.
